qml/Greeter/ButtonPrompt.qml

```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.4
+import Ubuntu.Components 1.3
+import "../Components"
+
+FocusScope {
+    id: root
+    objectName: "promptButton"
+    focus: true
+
+    property string text
+    property bool isSecret
+    property bool interactive: true
+    property bool loginError: false
+    property bool hasKeyboard: false
+    property string enteredText: ""
+    property bool inputFocus
```

Comment:
This seems unused

Reply:
Yes, it is unused, but not adding this will cause some warnings to show, this is because this file is loaded by the same loader as PinPrompt.qml and TextPrompt.qml. I could add a 'when' on the binding, but this will make the code more complex if we decide to add new login methods ( pattern, biometrical recognition... ).

Reply:
I see, but it seems to me that this variable in all login methods is either unused or not necessary: if the parent item is a FocusScope and the Input field has `focus: true` set, then the field will get the focus everytime the parent `root` element has the focus. So, instead of having an `inputFocus` property on the root element, you can just play with the root element's `focus` property, and if you need to explicitly move focus to it, you can just call `forceActiveFocus` on the root element.

Reply:
Ok, thank you! The only thing I still need is the isSecret property for the TextPrompt. It determines whether the input should be plain text or hidden. What to you suggest for that?